src/options.cpp

```diff
@@ -98,180 +98,152 @@ void long_usage(char const *arg0, bool verbose)
 {
     char const *const name = program_name(arg0);
 
-    printf("Usage:\n");
-    printf("\t%s [options] planet.osm\n", name);
-    printf("\t%s [options] planet.osm.{pbf,gz,bz2}\n", name);
-    printf("\t%s [options] file1.osm file2.osm file3.osm\n", name);
-    printf("\nThis will import the data from the OSM file(s) into a PostgreSQL "
-           "database\n");
-    printf("suitable for use by the Mapnik renderer.\n\n");
-
-    printf("%s", "\
-    Common options:\n\
-       -a|--append      Add the OSM file into the database without removing\n\
-                        existing data.\n\
-       -c|--create      Remove existing data from the database. This is the\n\
-                        default if --append is not specified.\n\
-       -l|--latlong     Store data in degrees of latitude & longitude.\n\
-       -m|--merc        Store data in proper spherical mercator (default).\n"
-#ifdef HAVE_GENERIC_PROJ
-                 "       -E|--proj num    Use projection EPSG:num.\n"
+    fmt::print(stdout, "\nUsage: {} [OPTIONS] OSM-FILE...\n", name);
+    std::fputs(
+        "\nImport data from the OSM file(s) into a PostgreSQL database.\n\n\
+Full documentation is available at https://osm2pgsql.org/\n\n",
+        stdout);
+
+    std::fputs("\
+Common options:\n\
+    -a|--append     Update existing osm2pgsql database with data from file.\n\
+    -c|--create     Import OSM data from file into database. This is the\n\
+                    default if --append is not specified.\n\
+    -O|--output=OUTPUT  Set output. Options are:\n\
+                    pgsql - Output to a PostGIS database (default)\n\
+                    flex - More flexible output to PostGIS database\n\
+                    multi - Multiple Custom Table Output to a PostGIS \n\
+                            database (deprecated)\n\
+                    gazetteer - Output to a PostGIS database for Nominatim\n\
+                    null - No output. Used for testing.\n\
+    -S|--style=FILE  Location of the style file. Defaults to\n\
+                    '" DEFAULT_STYLE "'.\n\
+    -k|--hstore     Add tags without column to an additional hstore column.\n",
+               stdout);
+#ifdef HAVE_LUA
+    std::fputs("\
+       --tag-transform-script=SCRIPT  Specify a Lua script to handle tag\n\
+                    filtering and normalisation.\n",
```

Comment:
I suggest to add a `(pgsql output only)`

Reply:
I have added this to the commit.